api/src/Page/Vstat.php

```diff
@@ -944,10 +944,15 @@ function _parameter_histogram()
         $beamlines = $this->_get_beamlines_from_type($this->ty);
         $bls = implode('\', \'', $beamlines);
 
+        $max_beam_size = 600;
```

Comment:
Is there any particular reason for having the i23 case be the default?

Reply:
It was so that the "Compare all beamlines" tickbox showed the larger beamsizes.